pkg/yqlib/format.go

```diff
@@ -112,7 +112,7 @@ func FormatStringFromFilename(filename string) string {
 		GetLogger().Debugf("checking file extension '%s' for auto format detection", filename)
 		nPos := strings.LastIndex(filename, ".")
```

Comment:
Not related to your changes, but why not using

filepath.Ext(filename)

Reply:
@ccoVeille good point! I like that `filepath.Ext` make it a bit more reliable, thanks!

Reply:
You are welcome